lib/helpers/is-interactive-element.js

```diff
@@ -13,10 +15,10 @@ const INTERACTIVE_TAG_NAMES = new Set([
   'label',
   'select',
   'textarea',
+  'video',
```

Comment:
This seems incorrect. `video` should only be added if `controls` attribute is present. See spec link / image above.

Reply:
Again, to AT, `video` means something, with or without controls. Lack of controls is a WCAG violation.